### PR TITLE
[vcpkg] ARM64 support

### PR DESCRIPTION
**Pull request by Mixaill**
Requires
- MSVS 2017 Update 4 Preview
- CMake 3.10 nightly build

### CODE REVIEW

File: scripts/cmake/vcpkg_configure_cmake.cmake [hunk 1]
```diff
@@ -78,14 +78,20 @@ function(vcpkg_configure_cmake)
...
+    elseif(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore" AND TRIPLET_SYSTEM_ARCH STREQUAL "arm" AND VCPKG_PLATFORM_TOOLSET MATCHES "v141")
         set(GENERATOR "Visual Studio 15 2017 ARM")
+    elseif(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore" AND TRIPLET_SYSTEM_ARCH STREQUAL "arm64" AND VCPKG_PLATFORM_TOOLSET MATCHES "v141")
+        set(GENERATOR "Visual Studio 15 2017")    
+        set(ARCH "ARM64")
     elseif(TRIPLET_SYSTEM_ARCH MATCHES "x86" AND VCPKG_PLATFORM_TOOLSET MATCHES "v141")
         set(GENERATOR "Visual Studio 15 2017")
     elseif(TRIPLET_SYSTEM_ARCH MATCHES "x64" AND VCPKG_PLATFORM_TOOLSET MATCHES "v141")
         set(GENERATOR "Visual Studio 15 2017 Win64")
     elseif(TRIPLET_SYSTEM_ARCH MATCHES "arm" AND VCPKG_PLATFORM_TOOLSET MATCHES "v141")
```
Comment (KindDragon) [inline scripts/cmake/vcpkg_configure_cmake.cmake]: Should not there be STREQUAL too?

Comment (Mixaill) [reply scripts/cmake/vcpkg_configure_cmake.cmake]: fixed

**Comment by Mixaill** [comment]
MSVS with ARM64 support was released at October 10, 2017.

Waiting for CMake 3.10.

**Comment by ras0219-msft** [comment]
Thanks for the PR!

I think this is pretty much perfect, aside from the waiting for CMake :). Since that's just a waiting game, I think I'll merge this now and we'll correct whatever is needed once CMake updates.

**Pull request merged**